apps/frontend/src/pages/onboarding/libs/components/onboarding-answer/onboarding-answer.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { Input } from "~/libs/components/components.js";
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+} from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type AnswerProperties<T extends FieldValues> = {
+	control: Control<T, null>;
+	name: FieldPath<T>;
+	options?: { label: string; value: string }[];
+};
+
+const OnboardingAnswer = <T extends FieldValues>({
+	control,
+	name,
+	options = [],
+}: AnswerProperties<T>): JSX.Element => {
+	return (
+		<div className={styles["onboarding-answer"]}>
+			<Input
+				control={control}
+				label=""
```

Comment:
We need to have label. If you don't want to display label, add `hasVisuallyHiddenLabel` and use `visually-hidden` class